### PR TITLE
Ms dict

### PR DESCRIPTION
**Pull request by Sjhunt93**

### CODE REVIEW

**Review by zeritte — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -613,8 +633,89 @@ Birdcache methods are:
       audience={"username": ["salih", "sam", "timo"]},
       soundTemplate="openup"
     )
+
+### `Lexi` resource <a name = "lexi"> </a>
+
+Lexi is a dictionary that will replace problematic words with phonetic spellings. For example,ensuring the city name `reading` is pronounced correctly. Dictionaries are split into languages and types. For example, the city `reading` would be of type `location` with language `en-gb` and in this case in the `UkCities69` dictionary.
```
Comment (GetOn4) [inline README.md:639]: Do we want to say that we using phonetic spelling?

**Review by GetOn4 — APPROVED** [review]
- LGTM

File: README.md (hunk 1 above)
Comment (Sjhunt93) [reply README.md:639]: Im not sure I guess this is PO question

Comment (GetOn4) [reply README.md:639]: @LindaKivine

**Review by zeritte — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -613,8 +633,89 @@ Birdcache methods are:
...
+    r = apiaudio.Script.create(scriptText=text)
+    scriptId = r["scriptId"]
+    
+    # preview the script in en-gb
+    preview = apiaudio.Script.preview(scriptId=scriptId, language="en-gb")
+    print(preview)
+    ```
+  - Response:
+    ```python
+    "The author of this repo has lived in two places in the UK, bude and [<!>bristol]<!>".
```
Comment (zeritte) [inline README.md:715]: just noticed the closing ] is wrongly placed

**Pull request merged**